src/models/article-sidebar.js

```diff
@@ -0,0 +1,20 @@
+import mongoose from 'mongoose';
+import slugs from 'slugs';
+
+
+
+const Schema = new mongoose.Schema({
+  name: {
+    type: String,
+    required: true,
+  },
+  sections: [{
+    type: String
+    required: true,
+
+}]
```

Comment:
Indent this so it's in line with its matching brace. You're also probably missing a comma after the `]`. Run `npm run lint` to catch style errors